api/core/providers/anthropic/anthropic_domain.py

```diff
@@ -281,6 +292,12 @@ def from_domain(cls, tool: DomainTool):
     # https://docs.anthropic.com/en/api/messages#body-tools
     tools: list[Tool] | None = None
 
+    class Thinking(BaseModel):
+        type: Literal["enabled"]  # 'disabled' is never used
```

Comment:
nit: default value then ?

Reply:
corrected in https://github.com/WorkflowAI/WorkflowAI/pull/671/commits/1ddfdd0a639692a04a4435896c42e7be8bfec68b